resources/assets/js/components/people/ContactInformation.vue

```diff
@@ -56,7 +56,7 @@
     </ul>
 
     <div v-if="addMode">
-      <form class="measure center">
+      <form class="measure center" v-on:submit.prevent>
```

Comment:
We could allow user to hit `Enter`, and just update this form
```suggestion
          <form class="measure center" v-on:submit.prevent="store">
```